spec/countries/MEX-spec.coffee

```diff
@@ -79,3 +79,24 @@ describe 'Mexico', ->
 
 			# Assert
 			expect(result).to.be.true
+
+		it 'all Mexico NDCs', ->
+			# Arrange
+			prefix  = "+52 "
+			suffix = " 111 2222"
+			suffix8 = "0 111 2222"
+
+			# Act
+			for ndc in Phone.countries['52'].nationalDestinationCode
+				if ndc.length is 2
+					number = prefix + ndc + suffix8
+				else
+					number = prefix + ndc + suffix
+
+				result = Phone.validate(number)
+				console.log(number, result)
+
+				# Assert
+				if !result
+					console.log 'NDC missing: ', ndc
```

Comment:
Ditto

Reply:
This is intended. The log will show missing NDCs along with the test results. 😄 
I've copied from this other test: https://github.com/vtex/front.phone/blob/master/spec/countries/NANP-spec.coffee#L14-L22

Reply:
Ok, makes sense.